src/controllers/calculateController.ts

```diff
@@ -0,0 +1,16 @@
+import express, { Request, Response, NextFunction } from "express";
+import { validateNumbers } from '../middelwares/validateNumbers';
+import {calculate} from '../services/calculate';
+
+const app = express();
+
+app.use(express.json());
+
+app.get("/calculate/:op", validateNumbers, calculate, (req, res) => {
+    res.status(200).json({ result:req.result });
+});
+
+const PORT = 3000;
+app.listen(PORT, () => {
```

Comment:
All of this code should be in the main file not here in the controller.
Please double-check this @ihebab99.